spec/fitbank_api/pix/payout_detail_spec.rb

```diff
@@ -0,0 +1,37 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'securerandom'
+
+RSpec.describe FitBankApi::Pix::PayoutDetail do
+  describe 'payout' do
+    let!(:payout) { build(:manual_payout) }
+
+    it 'can get payout detail' do
+      VCR.use_cassette('pix/payout_detail/manual_payout') do
+        response = payout.call
+        expect(response[:Success]).to eq('true')
+
+        @payout_id = response[:DocumentNumber].to_s
+      end
```

Comment:
Instead of using an instance variable, you can get the value from the VCR block:

```suggestion
      response = VCR.use_cassette('pix/payout_detail/manual_payout') do
        payout.call
      end

      expect(response[:Success]).to eq('true')

      payout_id = response[:DocumentNumber].to_s
```

Reply:
Done